public/css/styles-srl.css

```diff
@@ -28579,6 +28579,17 @@ a:visited {
   padding-bottom: 0 ;
```

Comment:
This shouldn't be effecting styles-srl.css or any other srl files. I feel like this code might be coming from a change in .scss elsewhere.

Reply:
@zakiya That's weird. The only SCSS file I changed was _header-trial.scss.  

I pulled and rebuild the latest development code on it's own and it makes the same changes to styles-srl.css.  Seems like somehow that got missed in the last development build.  Odd.

I made a branch of it and pushed for reference.  Could you please verify?  Maybe that could be merged first so this PR is cleaner?

https://github.com/Exygy/courtyard/pull/260/files

Reply:
I think that @evanmac123 and/or @ivanbueno-jcc just needs to confirm that the code is good and intended for SRL.

Reply:
Any component sass will affect both styles-srl.css and styles-trial.css.  Theme-specific sass files live here: https://github.com/Exygy/courtyard/tree/development/source/_patterns/00-protons